### PR TITLE
Jet menus and popovers for the dark gtk theme

### PR DESCRIPTION
**Pull request by Feichtmeier**
Continuation of https://github.com/ubuntu/yaru/pull/989

Much less code, same look.

### CODE REVIEW

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by clobrano** [comment]
Is it possible that toggled-inactive button in background still has the old pressed effect?

image

I also wanted to propose some adjustments in backdrop bg colors, but I'll do it after this PR

**Comment by Feichtmeier** [comment]
@clobrano 

image

It is a bit tricky, since the borders and shadows are rendered a bit differently in menus, context menus, csd menus and popovers but this way it works :) I renamed the variable, because I needed to change it for popvers only, so naming it popover border would be pointless

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

File: gtk/src/light/gtk-3.20/_common.scss [hunk 1]
```diff
@@ -2410,6 +2426,15 @@ popover.background {
     &:dir(ltr) { margin-right: 3px; }
     &:dir(rtl) { margin-left: 3px; }
   }
+
+  @if $variant=='dark' {
+    separator { background-color: darken($menu_border,12%); &:backdrop { background-color: transparent; } }
+    * { background-color: transparent; }
```
[the remark this answers is not in the document]
Comment (Feichtmeier) [reply gtk/src/light/gtk-3.20/_common.scss]: I fear yes. Ofc I could got through some of the widgets we know like calendar, scrolled window, etc. but 1) it is more code and 2) we may not get them all : /

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/yaru-pr1009`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/yaru-pr1009` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by Feichtmeier** [comment]
@madsrh could you please test this and share your opinion? :D

**Comment by madsrh** [comment]
> @madsrh could you please test this and share your opinion?

Sorry for the late reply. It looks good 👍 +1

**Pull request merged**